docs/upgrading/upgrading-from-30.md

```diff
@@ -1,14 +1,230 @@
+will be put in.
+
+### Moving Bolt 3 files into Bolt 5 installation
+Once you have installed the latest version of Bolt 5 move forward by copying relevant
+files of yout Bolt 3 site into the recently installed version 5.
+In the table below you can see an overview of the list of directories and files to be copied and  
+the location in the new version's structure.
+
+| Bolt 3                                | Bolt 5                                      |
+| ------------------------------------- | -------------------------------------------- |
+| General config directory `app/config`               | `config/bolt`                  |
+| Extensions config directory `app/config/extensions` | `config/extensions`            |
+| Templates directory `public/theme/YOUR_THEME_NAME`  | `public/theme/YOUR_THEME_NAME` |
+| Files directory `public/files`                      | `public/files`                 |
+| Thumbnails directory `public/thumbs`                | `public/thumbs`                |
+
+In case you have customized the default Bolt 3 `.htacces` file in the `public` directory,
+you will need to include the modification in the new Bolt 5 `public/.htaccess` file for your
+site to properly work.
+
+## Contenttype fields conversion
+Now that we have allocated the files in the right directories is time to get your `contenttypes.yml`
+file compatible with Bolt 5. Since new features has been added to fields and the definition of some Fields have
+slightly changed, these fields need to be converted  so Bolt 5 understands the type of field you are
+defining. It is not necessary to change all the fields but it is necessary to apply changes to the
+following Fields.
+
+#### Integer and Float fields
+The integer and the float fields are known in Bolt 3 like shown below:
+```yaml
+# Bolt 3 Integer field definition
+integerfield:
+    type: integer
+
+# Bolt 3 Float field definition
+floatfield:
+    type: float
+```
+These fields do no longer exist in Bolt 5. This types of fields have been combined into one Field called
+[Number](https://docs.bolt.cm/4.0/fields/number "Bolt documentation: Number field")
+
+It comes in with an option called `mode` where you can choose between `integer` or `float` to
+save that type of data in your Database.
+
+To migrate the data of an **Integer** field the new field definition will look like:
+```yaml
+# Bolt 5 integer Number field definition
+integerfield:
+    type: number
+    mode: integer
+```
+
+To migrate the data of an **Integer** field the new field definition will look like:
+```yaml
+# Bolt 5 float Number field definition
+floatfield:
+    type: number
+    mode: float 
+```
+
+### Repeater and Block fields
+The Reapeater and Block fields do no longer exist in Bolt 5. The Collection and Set field have been
+introduced to take over the role the other fields played. Even though the look and feel is pretty
+much the same as it was in earlier versions, the syntax to define these fields has differ a little bit.
```

Comment:
`has differ` -> `differs`

Reply:
This is changed and it makes more sense now